android/assets/jsons/translations/Brazilian_Portuguese.properties

```diff
@@ -1069,7 +1069,7 @@ defence vs ranged = defesa vs ataque à distancia
 [percentage] to unit defence = [percentage] a defesa da unidade
 Attacker Bonus = Bônus de Atacante
 Defender Bonus = Bônus de defesa
-Landing = Aterrissagem
+Amphibious = Aterrissagem
```

Comment:
I usually dont like doing this, as it gives no indication that translators need to update this translation.
Especially if it's only a single word and because it completely changes the meaning of it, this can be problematic and retranslating it from scratch doesn't take that long.
I'd have removed the word from the template file and then this new change would be distributed next version update.

Additionally, if I'm not mistaken 'amphibious' is the name of a promotion and should as such already be translated. Renaming these translations could then result in these strings having two different translations, of which the wrong one might be discarded when regenerating the translation files.